### DISCUSSION TITLE
Aggregate view of all workflows?

### DISCUSSION THREAD

**Discussion by pzybe** [community/community#26017] [Actions]
<p>* Is there some method via the UI by which a user can see the status of all of their Actions build jobs on one page?</p>
<p>* Is there some method via either the v3 or v4 API to make a query that will list the statuses for all Actions build jobs</p>
<p>if not:</p>
<p>are there plans to add these features at any point?</p>

**Comment by mscoutermarsh** [comment]
<p><em>&gt; Is there some method via the UI by which a user can see the status of all of their Actions build jobs on one page?</em></p>
<p>We have the Actions tab (/actions in your repo). What other info would you like to see on the page? I’m guessing more granular statuses for individual jobs?</p>
<p><em>&gt; Is there some method via either the v3 or v4 API to make a query that will list the statuses for all Actions build jobs</em></p>
<p>Not yet. There will be an API that you can grab this data from soon.</p>

**Comment by pzybe** [comment]
<p>hey <a class="mention" href="/mscoutermarsh">@mscoutermarsh</a> looking forward to seeing the info in the API.  I like the information page on each repo, what I’m asking for is something either user-level, or org-level that would essentially show the statuses for build jobs from <em>each</em> repo owned by that user or org.  This way, instead of having to travel to each repo’s /action tab, I can visit one page and get an overview of all my repo builds.</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>Ah got it. We have no plans for a UI like that at the moment. The API will also be by repository. There won’t be an endpoint for entire user/org, would have to do a few requests and aggregate them.</p>
<p>Thanks for sharing your usecase. Sorry we don’t have an exact solution. Will keep this in mind as we build out more org/user-wide features.</p>

**Comment by dstein64** [comment]
<p>I was interested in similar functionality.</p>
<p>I implemented a web page that shows an aggregated view of workflows and their statuses across a user’s repositories.</p>
<p>Here’s the link, in case this functionality would be useful to others.<br>
<a href="https://dstein64.github.io/workflows/" class="onebox" target="_blank" rel="nofollow noopener">https://dstein64.github.io/workflows/</a></p>
<p>And here’s a link to the source code:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/dstein64/workflows" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/541289/181094780-c085b7b7-a6a4-4115-910f-53a880f8599c.png" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/dstein64/workflows" target="_blank" rel="nofollow noopener">dstein64/workflows</a></h3>

<p>▶️ View the status of GitHub workflows. Contribute to dstein64/workflows development by creating an account on GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by chrisdrobison** [comment]
This would be a great feature.
